pandas/tests/frame/test_operators.py

```diff
@@ -530,6 +530,34 @@ def test_comp(func):
         test_comp(operator.ge)
         test_comp(operator.le)
 
+    @pytest.mark.parametrize("op", ["lt", "le", "gt", "ge", "eq", "ne"])
```

Comment:
Can you use the `compare_operators_no_eq_ne` pytest fixture instead?

Reply:
Thanks, @mroeschke - that was neat! Now using `compare_operators_no_eq_ne`.